Gemfile.lock

```diff
@@ -2,6 +2,7 @@ GEM
   remote: http://rubygems.org/
   specs:
     RedCloth (4.2.9)
+    RedCloth (4.2.9-x86-mingw32)
```

Comment:
I'm relatively new to git so bear with me. I have added this Thumb.db to my
global ignore and I ran "git rm cache */thumbs.db" which removed:

rm 'app/assets/img/Thumbs.db'
rm 'app/assets/img/blog/Thumbs.db'
rm 'app/assets/img/codelab/Thumbs.db'

However with the above comment on the gemfile.lock do you mean I should
just delete that line:

RedCloth (4.2.9-x86-mingw32)

From the file?

I'll fix up the wording on that other part you highlighted.

Regards,
Seth



[image: 3sparks llc] <contact@sethreid.co.nz>

Seth Reid | Web Consultant/Developer

0221705966 | contact@sethreid.co.nz

On 30 April 2015 at 06:27, Eddie Monge <notifications@github.com> wrote:

> In Gemfile.lock
> <https://github.com/yeoman/yeoman.io/pull/468#discussion_r29365238>:
>
> > @@ -2,6 +2,7 @@ GEM
> >    remote: http://rubygems.org/
> >    specs:
> >      RedCloth (4.2.9)
> > +    RedCloth (4.2.9-x86-mingw32)
>
> these shouldnt be checked in
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yeoman/yeoman.io/pull/468/files#r29365238>.
>